packages/typescript/README.md

```diff
@@ -79,6 +79,15 @@ Default: `null`
 
 A [minimatch pattern](https://github.com/isaacs/minimatch), or array of patterns, which specifies the files in the build the plugin should operate on. By default all `.ts` and `.tsx` files are targeted.
 
+### `filterRoot`
+
+Type: `String` | `Boolean`<br>
+Default: `true`
+
+Sets the `resolve` parameter for [createFilter](https://github.com/rollup/plugins/tree/master/packages/pluginutils#createfilter). By default, it will resolve against the `rootDir` set in the TS config file however you can set this to change which files are filtered out by the plugin.
```

Comment:
These docs are a little difficult to read if you're not familiar with Rollup's internals. I'd prefer something based on `createFilter`'s docs, talking about how this option sets the base directory for `include` and `exclude`.

i.e.:

>Optionally resolves the `include` and `exclude` patterns against a directory other than `process.cwd()`. If a `String` is specified, then the value will be used as the base directory. Relative paths will be resolved against `process.cwd()` first. If `false`, then the patterns will not be resolved against any directory. 
>
> By default, patterns resolve agaisnt the `rootDir` set in your TS config file.
>
>This can fix plugin errors when parsing files outside the current working directory (`process.cwd()`)

Reply:
Updated